source/common/secret/secret_manager_impl.cc

```diff
@@ -34,6 +34,8 @@ void SecretManagerImpl::addStaticSecret(const envoy::api::v2::auth::Secret& secr
     }
     break;
   }
+  // envoy::api::v2::auth::Secret::TypeCase::kTrustedCa is not supported, as static trusted ca
+  // may not be very useful.
```

Comment:
Why? It would be useful to provide public trusted CAs in one static and refer them?

Reply:
Well, on second thought.  we could add static secret for trusted CA.

Reply:
I have added back static secret for trusted CA. Will add tests later.